### PR TITLE
Implement welcome messages

### LINKED ISSUE #16: Welcome messages
We should facilitate for setting [welcome messages](https://developers.facebook.com/docs/messenger-platform/send-api-reference#welcome_message). Maybe something like `Facebook::Messenger::Welcome.set [...]`?

### PR DESCRIPTION
**Pull request by jgorset**
Fixes #16

### CODE REVIEW

File: lib/facebook/messenger/welcome.rb [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+module Facebook
+  module Messenger
+    # This module handles setting welcome messages.
+    #
+    # https://developers.facebook.com/docs/messenger-platform/send-api-reference#welcome_message_configuration
+    module Welcome
+      include HTTParty
+
+      base_uri 'https://graph.facebook.com/v2.6/me'
```
Comment (mengqing) [inline lib/facebook/messenger/welcome.rb]: I think the API endpoint for welcome message is `https://graph.facebook.com/v2.6/<PAGE_ID>/thread_settings?access_token=<PAGE_ACCESS_TOKEN>`

https://developers.facebook.com/docs/messenger-platform/send-api-reference#welcome_message_configuration

**Comment by jgorset** [comment]
Yeah! `me` is a shorthand for the page that the given access token belongs to, though. That way, we never even have to ask which page it is.

File: lib/facebook/messenger/welcome.rb (hunk 1 above)
Comment (jgorset) [inline lib/facebook/messenger/welcome.rb]: Yeah! me is a shorthand for the page that the given access token belongs to, though. That way, we never even have to ask which page it is.

**Comment by sorich87** [comment]
Where was this merged? I can't see it in master.

**Comment by jgorset** [comment]
Neither can I… that's pretty weird.

**Comment by jgorset** [comment]
I still had the branch locally, so I merged it to master (again) and released version 0.7.0. Thanks @sorich87!

**Comment by sorich87** [comment]
Thank you!

**Pull request merged**